lisa/tools/reboot.py

```diff
@@ -127,3 +133,34 @@ def reboot(self, time_out: int = 300) -> None:
                 raise LisaException(
                     "timeout to wait reboot, the node may stuck on reboot command."
                 )
+
+
+class WindowsReboot(Reboot):
+    @property
+    def command(self) -> str:
+        return "powershell"
+
+    def reboot(self, time_out: int = 300) -> None:
+        self.node.tools[PowerShell].run_cmdlet("Restart-Computer -Force")
+
+        # wait for nested vm ssh connection to be ready
+        from lisa.node import RemoteNode
+
+        remote_node = cast(RemoteNode, self.node)
+        self._log.debug(
+            f"checking if SSH port {remote_node.public_port} is reachable "
+            f"on {remote_node.name}..."
+        )
+
+        # wait for ssh port to be ready
+        connected, _ = wait_tcp_port_ready(
+            address=remote_node.public_address,
+            port=remote_node.public_port,
+            log=self._log,
+            timeout=3,
```

Comment:
Does the timeout is enough to wait it boot up? How to make sure the node is rebooted actually, instead of stuck before reboot? Refer to Linux to check if the uptime is changed.